server/service/src/settings.rs

```diff
@@ -36,3 +38,46 @@ pub fn is_develop() -> bool {
     // debug_assertions is the recommended way to check if we are in 'dev' mode
     cfg!(debug_assertions)
 }
+
+#[derive(serde::Deserialize, Clone)]
+pub enum LogMode {
+    Console,
+    File,
+}
+
+#[derive(serde::Deserialize, Clone)]
+pub enum Level {
+    Off,
+    Error,
+    Warn,
+    Info,
+    Debug,
+    Trace,
+}
+
+impl From<Level> for LevelFilter {
+    fn from(level: Level) -> Self {
+        match level {
+            Level::Off => LevelFilter::Off,
+            Level::Error => LevelFilter::Error,
+            Level::Warn => LevelFilter::Warn,
+            Level::Info => LevelFilter::Info,
+            Level::Debug => LevelFilter::Debug,
+            Level::Trace => LevelFilter::Trace,
+        }
+    }
+}
+
+#[derive(serde::Deserialize, Clone)]
+pub struct LoggingSettings {
+    /// Console (default) | File
+    pub mode: LogMode,
```

Comment:
Do we need a Console + File mode ?

Reply:
I did wonder about that! will have a quick try and see if I can get that working..